dojo/tools/bearer_cli/parser.py

```diff
@@ -0,0 +1,55 @@
+import json
+from dojo.models import Finding
+
+
+class BearerParser(object):
+    """
+    Bearer CLI tool is a SAST scanner for multiple languages
+    """
+
+    def get_scan_types(self):
+        return ["Bearer CLI"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "Bearer CLI"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Bearer CLI report file can be imported in JSON format (option -f json)."
+
+    def get_findings(self, file, test):
+        data = json.load(file)
+
+        items = list()
+        dupes = set()
+
+        for content in data:
+            severity = content.capitalize()
+            for bearerfinding in data[content]:
+                print(bearerfinding)
```

Comment:
```suggestion
```

Reply:
Sorry for this. It's removed now. Thanks!